src/data/accessibility-quiz.ts

```diff
@@ -334,16 +343,32 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete"
   },
   {
-    Question:
-      "What is the minimum contrast ratio needed, according to the WCAG?",
+    Question: "What is the function of the `<contentinfo>` ARIA role?",
+    Answer: "displays content at that bottom of a website",
+    Distractor1: "displays content",
+    Distractor2: "displays a list",
+    Distractor3: "creates content",
+    Explanation: "The `<contentinfo>` role is used describe the information that is presented at the bottom of the website, such as copyright statements, links to help people navigate the website, etc. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role",
+  },
+  {
+    Question: "What is the ARIA role, `<comment>`used for?",
+    Answer: "To describe a reaction or emotion in some content",
+    Distractor1: "To provide notes about certain lines of code",
+    Distractor2: "To share an opinion",
+    Distractor3: "To critique a piece of text",
+    Explanation: "If a website has a testimonial section, the `<comment>` role helps describe this information to people who use screen readers.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/comment_role",
+  },
+   { Question: "What is the minimum contrast ratio needed, according to the WCAG?",
```

Comment:
it looks like we have a formatting issue here
please fix this so it aligns like the rest of the objects